metadata-ingestion/src/datahub/ingestion/source/mode.py

```diff
@@ -1523,20 +1572,9 @@ def emit_chart_mces(self) -> Iterable[MetadataWorkUnit]:
             for report in reports:
                 report_token = report.get("token", "")
 
-                if report.get("imported_datasets"):
-                    # The connector doesn't support imported datasets yet.
-                    # For now, we just keep this in the report to track what we're missing.
-                    imported_datasets = [
-                        imported_dataset.get("name") or str(imported_dataset)
-                        for imported_dataset in report["imported_datasets"]
-                    ]
-                    self.report.dropped_imported_datasets.setdefault(
```

Comment:
remove dropped_imported_datasets from the report

Reply:
exactly - we don't need `dropped_imported_datasets` anymore, so we should remove it in all places